Makefile

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env make -f
+# This Makefile assumes an activated Python virtual environment.
+
+.PHONY:	all
+all:	dist
+
+dist:	dist/.dist
+dist/.dist:
+		python setup.py bdist
+		python setup.py sdist
+		touch dist/.dist
+
+.PHONY:	check
+check:
+		pylint src/ tests/
+		mypy src/ tests/
```

Comment:
Alternatively, we could consider calling `pre-commit run --all-files` here.

Reply:
How about `pytest`? And again we need to decide between `pre-commit` and the CI tests?

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/74/commits/139ef23b1619e8b00394129043c3c621018f54ed.